mldsa/polyvec.c

```diff
@@ -162,20 +162,25 @@ void polyvecl_reduce(polyvecl *v)
   }
 }
 
-void polyvecl_add(polyvecl *w, const polyvecl *u, const polyvecl *v)
+/* Reference: `polyvec_add()` in the reference implementation @[REF].
```

Comment:
it's also called `polyvecl_add` in the reference implementation. Also you did not rename it, so there is no need to state it here.